alf/algorithms/mbrl_algorithm.py

```diff
@@ -170,6 +175,23 @@ def _expand_to_population(self, data, population_size):
         data_population = torch.repeat_interleave(data, population_size, dim=0)
         return data_population
 
+    def _expand_to_particles(self, inputs):
+        # inputs [B, n, d]
+        if inputs.ndim == 2:
```

Comment:
when `inputs.ndim` is not 2?

Reply:
This check is unnecessary. Updated this function.